arches/app/models/migrations/11955_set_graphs_active.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 5.2.2 on 2025-06-04 14:38
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("models", "12002_percent_sign_fix"),
+    ]
+
+    def set_active_true(apps, schema_editor):
+        GraphModel = apps.get_model("models", "GraphModel")
+        # Select published graphs without any draft graphs.
+        GraphModel.objects.filter(
+            # "graphmodel" is the default related_name for GraphModel.source_identifier
```

Comment:
If you're game, I could just fix this in this PR e.g. `related_name="draft"` on the models in question.